### ISSUE TITLE
C implimentaion of this progam

### ISSUE THREAD

**Issue by HindolKarfarma-learn** [codewithnihar2027/guess-the-number#2]
Please dont close this issue until the the side pull request is done.

It is common equate to ask repository's owner before starting a pull request.
pull request will ask the owner to add a change to the code base,

here i want start implementing the C implementation of this game.
i will start with adding the guess_the _number.c file and add header like #include <std.io>
@codewithnihar2027 please comment if you want this feature

**Comment by codewithnihar2027** [comment]
yes you can implement this

**Comment by HindolKarfarma-learn** [comment]
there is a  new issue, there is no } for the rng function. 
please add that and commit it, it was my fault
